examples/Image Segmentation/ios/Podfile

```diff
@@ -0,0 +1,20 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'ImageSegmenter' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  pod 'TensorFlowLiteTaskVision'
+
+  target 'ImageSegmenterTests' do
```

Comment:
I deleted these targets and am good to go, so we either need to correct the target names or remove these for building.

Reply:
I missed it when I deleted the UI tests folder